ooni/ui/web/client/index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="black">
+  </head>
+  <body ng-app="app" ng-strict-di ng-cloak>
+
+    <app>
+      Loading...
```

Comment:
Well that is actually displayed for some seconds while the JS resources are fetched and loaded. I agree though that we could perhaps register some exception handlers on the loading of the assets and if it gets triggered also show some instructions on how to set it up.

I plan however on adding the source for the web app to the repo every time a stable release is tagged so that people can also use it from git builds.